nbs/13a_learner.ipynb

```diff
@@ -3583,7 +3583,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In practice, we get the predictions `n` times with the transforms of the training set and average those. The final predictions are `(1-beta)` multiplied by this average + `beta` multiplied by the predictions obtained with the transforms of the dataset. Set `beta` to `None` to get a tuple of the predictions and tta results. You can also use the maximum of all predictions instead of an average by setting `use_max=True`.\n",
+    "In practice, we get the predictions `n` times with the transforms of the training set and average those. The final predictions are `(1-beta)` multiplied by this average + `beta` multiplied by the predictions obtained with the transforms of the dataset. Set `beta` to `None` to get a tuple of the predictions and the results. You can also use the maximum of all predictions instead of an average by setting `use_max=True`.\n",
```

Comment:
I believe that this should be left as "tta"

Reply:
Sorry about that, will change now!